mslice/models/cut/matplotlib_cut_plotter.py

```diff
@@ -33,35 +35,36 @@ def plot_cut_from_xye(self, x, y, e, x_units, selected_workspace, plot_over, leg
         plt.xlabel(self._getDisplayName(x_units, self._cut_algorithm.getComment(selected_workspace)), picker=picker)
         plt.ylabel(INTENSITY_LABEL, picker=picker)
         plt.autoscale()
+        plt.gcf().canvas.manager.add_cut_plot(self)
         if not plot_over:
             plt.gcf().canvas.manager.update_grid()
-        self.axes.draw_artist(self.canvas.figure.get_children()[1])
-        self.canvas.blit(self.canvas.figure.gca().clipbox)
+        plt.gcf().canvas.draw()
+        plt.gca().draw_artist(plt.gcf().canvas.figure.get_children()[1])
+        plt.gcf().canvas.blit(plt.gcf().canvas.figure.gca().clipbox)
```

Comment:
Why do we need `draw()` here? `draw_artist` and `blit` were used to avoid calling `draw()` (which seems quite slow).